### ISSUE TITLE
Layout as a Service does not respect UUID

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#14250]
In 5.x we allow the same container to be used multiple times on a page.  The issue is that when you hit and retrieve the page via LaaSy that has multiple of the same container, it only returns one and that one unrendered (because is has no concept of the uuid).

To reproduce, edit the `/about-us/index` page and add the same container in multiple places.  Then add content to all those containers.  The page renders fine, but when you call the page API, `/api/v1/page/render/about-us/index` you get a single empty container.

Basically, this example is broken:

https://dotcms.com/blog/post/beyond-headless-content-layout-as-a-service-in-dotcms

------

## Acceptance criteria in the frontend
@fmontes: Content should remain if we change the template for another with the same containers

**Comment by fmontes** [comment]
If you add the same container in the sidebar and in a box the content is duplicated.

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/14492

**Comment by fmontes** [comment]
When you remove the sidebar with a content already existing in the page, the content from the container in the sidebar render in the container left in the page.

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/14569

**Comment by alfredo-dotcms** [comment]
there's a bug when added side nav bar container, steps:
1) reset tomcat
2) edit page --> about index
3) layout
4) add container
5) add LARGE CONTAINER
6) CONTENT
7) Add Content --> ABOUT QUEST
8) Actions --> Save/Publish
9) Layout
10) Add Right Sidenav bar
11) add LARGE CONTAINER
12) CONTENT

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/14677

**Comment by bryanboza** [comment]
Fixed, tested in the last master and now I'm unable to reproduce
